docs/troubleshooting/remote-troubleshooting.md

```diff
@@ -0,0 +1,282 @@
+Troubleshooting Remote HTCondor-CEs
+===================================
+
+Since HTCondor-CE is built on top of HTCondor, it's possible to perform quite a bit of troubleshooting from a remote
+client with access to the HTCondor command-line tools.
+For testing end-to-end resource request submission or remote interactive-like access, a grid operator will need access
+to a host with the `htcondor-ce-client` installed.
+This document outlines the steps that a grid operator can perform in order to troubleshoot a remote HTCondor-CE.
+
+Verifying Network Connectivity
+------------------------------
+
+Before performing any troubleshooting of the remote HTCondor-CE service, it's important to verify that the HTCondor-CE
+can be contacted on its HTCondor-CE port (default: `9619`) at the specified fully qualified domain name (FQDN).
+
+### Verifying DNS ###
+
+!!! tip "Reverse DNS and GSI authentication"
+    GSI authentication requires that the HTCondor-CE host has a reverse DNS record but that record is not required to
+    match the forward DNS record! For example, if you have an `A` record `htcondor-ce.chtc.wisc.edu -> 123.4.5.678`, a
+    `PTR` of `123.4.5.678 -> chtc.wisc.edu` would satisfy the GSI authentication requirement.
```

Comment:
Really? That's pretty bizarre. I wonder what the security implications are...

Reply:
It's a bug, pure-and-simple.  PTRs are irrelevant to the security model.

I _think_ with HTCondor 8.9.9, we no longer need a PTR in any situation.

Reply:
> I think with HTCondor 8.9.9, we no longer need a PTR in any situation.

Yup but it requires a client-side upgrade so we're not out of the woods yet with the PTR requirement.